client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -46,7 +47,6 @@ import useValidCheckoutBackUrl from 'calypso/my-sites/checkout/composite-checkou
 import { leaveCheckout } from 'calypso/my-sites/checkout/composite-checkout/lib/leave-checkout';
 import { prepareDomainContactValidationRequest } from 'calypso/my-sites/checkout/composite-checkout/types/wpcom-store-state';
 import useCartKey from 'calypso/my-sites/checkout/use-cart-key';
-import { useDispatch as useReduxDispatch, useSelector } from 'calypso/state';
```

Comment:
I think we actually want to use these from `calypso/state` to preserve Type safety as per this request: p4TIVU-azW-p2

Reply:
Oh ok, this was done when I rebased, I didn't realize that difference. I'll get that fixed up!